kaniko.go

```diff
@@ -82,13 +95,32 @@ func (p Plugin) Exec() error {
 		cmdArgs = append(cmdArgs, fmt.Sprintf("--cache-ttl=%d", p.Build.CacheTTL))
 	}
 
+	if p.Build.DigestFile != "" {
+		cmdArgs = append(cmdArgs, fmt.Sprintf("--digest-file=%s", p.Build.DigestFile))
+	}
+
 	cmd := exec.Command("/kaniko/executor", cmdArgs...)
 	cmd.Stdout = os.Stdout
 	cmd.Stderr = os.Stderr
 	trace(cmd)
 
 	err := cmd.Run()
-	return err
+	if err != nil {
+		return err
+	}
+
+	if p.Build.DigestFile != "" {
+		content, err := ioutil.ReadFile(p.Build.DigestFile)
+		if err != nil {
+			fmt.Fprintf(os.Stderr, "%s\n", err)
```

Comment:
Do not log this line since returned error is logged in the main.go

Use: `return errors.Wrap(err, "failed to read digest file contents at path: %s", p.Build.DigestFile)`

Reply:
Im not returning those errors, if i return error plugin exec will be marked as failed.

Reply:
Update error message as suggested in previous comment

Reply:
done